src/xdotoolify.js

```diff
@@ -230,7 +230,18 @@ _Xdotoolify.prototype.checkUntil = function(f, ...rest) {
   });
   return this;
 };
-_Xdotoolify.prototype.mousemove = function(selector, relpos, twoStep, timeout) {
+_Xdotoolify.prototype.checkNothing = function() {
+  this._addOperation({
+    type: 'skip'
+  });
+  return this;
+};
+_Xdotoolify.prototype.mousemove = function(
+  selector,
```

Comment:
this, and every other publicly-visible action, should just call `_mousemove(selector, relpos, twoStep, timeout, true)`. Copypasta is very painful to maintain.

Reply:
Changed. Now `click` calls `_click` with `checkAfter = true` and so on.